connect/0.6/Dockerfile.local

```diff
@@ -18,6 +19,7 @@ RUN mkdir $KAFKA_CONNECT_PLUGINS_DIR/debezium
 COPY debezium-connector-mysql-$DEBEZIUM_VERSION-SNAPSHOT-plugin.tar.gz $KAFKA_CONNECT_PLUGINS_DIR/debezium-mysql-plugin.tar.gz
 COPY debezium-connector-mongodb-$DEBEZIUM_VERSION-SNAPSHOT-plugin.tar.gz $KAFKA_CONNECT_PLUGINS_DIR/debezium-mongodb-plugin.tar.gz
 COPY debezium-connector-postgres-$DEBEZIUM_VERSION-SNAPSHOT-plugin.tar.gz $KAFKA_CONNECT_PLUGINS_DIR/debezium-postgres-plugin.tar.gz
-RUN  tar -xzf $KAFKA_CONNECT_PLUGINS_DIR/debezium-mysql-plugin.tar.gz -C $KAFKA_CONNECT_PLUGINS_DIR/debezium --strip 1 &&\
-     tar -xzf $KAFKA_CONNECT_PLUGINS_DIR/debezium-mongodb-plugin.tar.gz -C $KAFKA_CONNECT_PLUGINS_DIR/debezium --strip 1 &&\
-     tar -xzf $KAFKA_CONNECT_PLUGINS_DIR/debezium-postgres-plugin.tar.gz -C $KAFKA_CONNECT_PLUGINS_DIR/debezium --strip 1
+
+RUN for CONNECTOR in {mysql,mongodb,postgres}; do \
+    tar -xzf $KAFKA_CONNECT_PLUGINS_DIR/debezium-$CONNECTOR-plugin.tar.gz -C $KAFKA_CONNECT_PLUGINS_DIR/debezium; \
```

Comment:
Yes, it works. But I prefer to have plaugins isolated if possible. So far we have no dependency conflicts but it may change in the future.